### DISCUSSION TITLE
Wiki changes are done!

### DISCUSSION THREAD

**Discussion by jones-michael-s** [automaticanalysis/automaticanalysis#329] [General]
So I've finished updating the aa wiki. I think the results are very much improved. The changes are a mix of new additions (my tutorial, and an FAQ) plus some minor (and no so minor) edits of the previous wiki pages to make them current and a bit easier to follow. I also integrated material from the Getting Started pages on automaticanalysis.github.io. Finally, I also added a module development guide I wrote for our lab as an Appendix.

I'm still finding typos here and there but I think we now have a solid body of up-to-date documentation moving forward.

I didn't make any changes to automaticanalysis.github.io (yet). As we discussed, this site would be good for announcements and aa news, especially now that the wiki is in reasonable shape. The only change needed there right now is to remove the "Getting Started" page and any references to it. (Perhaps we could also add an announcement that the wiki has been overhauled :-)

It looks like a pretty simple change but I hesitate because whereas Github provides a nice WYSIWYG editor for the wiki, the news site uses tools I'm not familiar with (ruby and gem and jekyll). I _think_ I see how to make the file changes in the repo but I don't want to break the site (also I'm unclear if commits there trigger the aa CI runners, which we don't really need).

 If you have time to make the change that works for me.  Otherwise I'll take a swing at it and see what happens...

**Comment by tiborauer** [comment]
It does look very much improved to me! It looks more modern and better organised. It is great to have your tutorial in this format which is easy to browse and read on all devices. I love the module development guide. It is almost the essence of aa, as it covers all aspects. :)

There is only one thing I do miss, however, especially in contrast with the website (automaticanalysis.github.io): the links between the parts. For example, [when you speak about the parameter file in the tutorial](https://github.com/automaticanalysis/automaticanalysis/wiki/Tutorial#creating-your-aa-parameter-file), it could reference the [Parameter File Setup section](https://github.com/automaticanalysis/automaticanalysis/wiki/Parameter-File-Setup). It would make the documentation more organic and motivate and guide the users for further reading and understanding.
I also have one small comment: I'd rather omit discussing parts we are not sure about or ignored (e.g. `'domain'`, `'whentorun'`, `'description'`, `'summary'` in the module body), because they are most likely to be removed, and we do not want users and developers build upon or confused by them.

The website automaticanalysis.github.io is not included in CI, because it is "live" in the sense that any changes are applied almost instantly. The repo description contains information about how to render the website locally, i.e. before committing any changes.
